src/nvim/macros.h

```diff
@@ -198,6 +198,16 @@
 # define IO_COUNT(x)  (x)
 #endif
 
+#define xfree(ptr) \
```

Comment:
Shouldn't this be `XFREE` so it's a) clear it is a macro and b) _actually_ forces people to handle the merge properly (as you hinted at initially).

Reply:
yeah, it was WIP. But I'm retreating anyway :)